scripts/prepare.sh

```diff
@@ -24,3 +24,41 @@ fi
 if ! type mockgen >/dev/null 2>&1; then
   go get github.com/golang/mock/mockgen
 fi
+
+function Get_PM_Name()
+{
+  PM=''
+  if grep -Eqii "CentOS" /etc/issue || grep -Eq "CentOS" /etc/*-release; then
+    DISTRO='CentOS'
+    PM='yum'
+  elif grep -Eqi "Red Hat Enterprise Linux Server" /etc/issue || grep -Eq "Red Hat Enterprise Linux Server" /etc/*-release; then
+    DISTRO='RHEL'
+    PM='yum'
+  elif grep -Eqi "Aliyun" /etc/issue || grep -Eq "Aliyun" /etc/*-release; then
+    DISTRO='Aliyun'
+    PM='yum'
+  elif grep -Eqi "Fedora" /etc/issue || grep -Eq "Fedora" /etc/*-release; then
+    DISTRO='Fedora'
+    PM='yum'
+  elif grep -Eqi "Debian" /etc/issue || grep -Eq "Debian" /etc/*-release; then
+    DISTRO='Debian'
+    PM='apt-get'
+  elif grep -Eqi "Ubuntu" /etc/issue || grep -Eq "Ubuntu" /etc/*-release; then
+    DISTRO='Ubuntu'
+    PM='apt-get'
+  elif grep -Eqi "Raspbian" /etc/issue || grep -Eq "Raspbian" /etc/*-release; then
+    DISTRO='Raspbian'
+    PM='apt-get'
+  else
+    DISTRO='unknow'
+  fi
```

Comment:
please consider macos system

Reply:
1、Commits is rebased.
2、Script is modified to be compatible with MacOS now.